lib/routes/account.js

```diff
@@ -1646,11 +1668,46 @@ module.exports = function (
             isVerified = isVerified && sessionVerified
           }
 
-          return {
-            email: sessionToken.email,
-            verified: isVerified,
-            sessionVerified: sessionVerified,
-            emailVerified: emailVerified
+          if (! isVerified) {
+            return bounces.check(sessionToken.email)
```

Comment:
This is something we should definitely call out in the deploy doc when this goes to release - this is a second hit to the DB on our most heavily-trafficked route, and almost all requests polling this status endpoint will have `isVerified=false`.

I don't honestly know whether this additional read load on the DB will be acceptable.... /cc @jrgm @jbuck 

This might argue in favour of us reifying the bounce state of an email as some sort of "status" field on the main accounts/emails table that we can query as part of checking the sessionToken, rather than recalculating it every time by slurping in a bunch of records from the db.  (See e.g. `sessionTokenWithDevice` for an example of where we do this already in order to reduce number of requests made to the db).